site/en/posts/iwd-2022-shree-bose-piper.md

```diff
@@ -0,0 +1,66 @@
+---
+title: 'Helping students fall in love with building technology - Shree Bose of Piper'
```

Comment:
```suggestion
title: 'Helping students fall in love with building technology with Piper's Shree Bose'
```

Just a slight rewording to align with how Julia's article's title was written

Reply:
Changed!